pkg/cmd/server/api/types.go

```diff
@@ -8,8 +8,8 @@ import (
 
 const (
 	FeatureBuilder    = `Builder`
-	FeatureS2I        = `S2I Builder`
-	FeatureWebConsole = `Web Console`
+	FeatureS2I        = `S2IBuilder`
```

Comment:
ok, then do the minimal impl needed to alias these two flags, since we won't add any new flags with spaces